sos/plugins/kubernetes.py

```diff
@@ -22,33 +22,75 @@ class kubernetes(Plugin, RedHatPlugin):
     """Kubernetes plugin
     """
 
-    option_list = [("podslog", "capture logs for pods", 'slow', False)]
+    packages = ('kubernetes',)
```

Comment:
Actually we should add `atomic-openshift-master`, which depends on `atomic-openshift` (but that one also gets pulled into nodes, and we don't need the kubernetes plugin auto-enabled there).